config/policies.tf

```
@@ -0,0 +1,4 @@
+resource "commonfate_policyset" "example" {
+  id   = "example"
```

Let's rename this `default` (and rename the policy file etc) rather than `example`. Example indicates that they need to change it whereas this should be a safe default they can use for production.

```suggestion
  id   = "default"
```